packages/manager/modules/pci/src/projects/project/kubernetes/service.js

```diff
@@ -77,16 +80,20 @@ export default class Kubernetes {
       .catch((error) => (error.status === 400 ? false : Promise.reject(error)));
   }
 
-  createCluster(projectId, name, region, version, nodepool) {
+  createCluster(projectId, name, region, version, privateNetworkId, nodepool) {
     if (nodepool.antiAffinity) {
       set(nodepool, 'maxNodes', ANTI_AFFINITY_MAX_NODES);
     }
-    return this.$http.post(`/cloud/project/${projectId}/kube`, {
+    const options = {
       region,
       name,
       version,
       nodepool,
-    });
+    };
```

Comment:
```suggestion
       ...privateNetworkId ? { privateNetworkId } : {}
    };
```
to avoid a `id` but nitpicking

Reply:
Fixed.